lib/counter_culture.rb

```diff
@@ -13,7 +13,13 @@ module ActiveRecord
 
     module ClassMethods
       # this holds all configuration data
-      attr_reader :after_commit_counter_cache
+      def after_commit_counter_cache
+        config = @after_commit_counter_cache
+        if !config && superclass.respond_to?(:after_commit_counter_cache)
+          config = superclass.after_commit_counter_cache
+        end
+        config
+      end
```

Comment:
Yeah, the sub-class config _would_ clobber the superclass. But it's not 100% clear to me what the correct behavior should actually be.

I agree that `@@` might be the way to go. I tried that route, unfortunately I couldn't get it to work. I think I'll give that another shot, though, as it really seems to be what I'm looking for here.